src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -260,48 +306,19 @@ private static void init(){}
         public static final TagKey<Item> GEMS_PRISMARINE = tag("gems/prismarine");
```

Comment:
`enchnating_fuels` is another tag that should be left under `forge`/`neoforge`. Its not changed in the diff so Github Won't let me comment on it directly.